content/updates/2019-09-06-depreecation-policy.md

```diff
@@ -0,0 +1,38 @@
++++
+date = "2019-08-26"
+title = "For Humans and Agencies: The cloud.gov Deprecation Policy" 
++++
+
+# For Humans and Agencies: The cloud.gov Deprecation Policy
+
+We'd like to talk about two aspects of being a hosting provider: consistency and predictability. In the context of being a hosting provider, consistentcy is the ability to do something the same way every time, and predictability means you can count on something to happen in a very specific way. We have been neither consistent nor predictable, and we'd like to talk about how things are changing and heading towards a positive direction.
```

Comment:
>We have been neither consistent nor predictable
I think we should be direct about our faults, but I think the way this is worded is overly critical and binary. Maybe something more like:
> There are several places where we've been inconsistent or unpredictable
or
>We've lacked consistency and predictability in some ways

Reply:
Seconding this, especially since the customer view of cloud.gov doesn't necessarily include the massive turnover we've had as a team.